vendor/bat-native-ledger/src/bat_contribution.cc

```diff
@@ -975,7 +976,7 @@ void BatContribution::VotePublishers(
   ledger_->AddReconcileStep(
       viewing_id,
       braveledger_bat_helper::ContributionRetry::STEP_FINAL);
-
+  ledger_->BroadcastReconcile(false);  // PHASE ONE COMPLETE
```

Comment:
I think this one can be problematic as one time tip can set this to false and then you lose track about auto contribute process.

Reply:
This is problematic 'as-is' even without losing track. We may have to block rewards clear until after phase two. Getting weird character counter and date results if we release early. Aside from that, I'll find a way to check for other 'in progress'es

Reply:
Fixed. Implemented counters on bat_contribution to update correctly.